site/docs/v1/tech/apis/two-factor.adoc

```diff
@@ -415,6 +365,56 @@ The value provided in the [field]#twoFactorTrustId# on the request.
 include::docs/src/json/two-factor/status/response.json[]
 ----
 
+== Send a Multi-Factor Code During Login or Step Up
```

Comment:
This section did not actually change. I just moved it down after `Retrieve Multi-Factor Status` to match the ToC.